mindsdb/integrations/handlers/mindsdb_inference/mindsdb_inference_handler.py

```diff
@@ -0,0 +1,93 @@
+import os
+import pandas as pd
+import openai
+from typing import Dict, Optional
+from mindsdb.integrations.handlers.openai_handler import Handler as OpenAIHandler
+from mindsdb.integrations.utilities.handler_utils import get_api_key
+from mindsdb.utilities import log
+
+logger = log.getLogger(__name__)
+MINDSDB_INFERENCE_BASE = 'https://llm.mdb.ai/'
+
+
+class MindsDBInferenceHandler(OpenAIHandler):
+    """
+    Handler for MindsDB Inference Endpoints
+    """
+
+    name = 'mindsdb_inference'
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.base_api = MINDSDB_INFERENCE_BASE
+        self.default_model = 'gpt-3.5-turbo'
+        self.default_mode = 'default'
+
+    @staticmethod
+    def _get_api_key(args, engine_storage):
```

Comment:
This method seems to be redundant - since below you're using the `get_api_key` method.

Reply:
Good catch, removing it